pgbouncer/datadog_checks/pgbouncer/pgbouncer.py

```diff
@@ -33,14 +34,21 @@ class PgBouncer(AgentCheck):
             ('database', 'db'),
         ],
         'metrics': [
-            ('total_requests',       ('pgbouncer.stats.requests_per_second', RATE)),
+            ('total_requests',       ('pgbouncer.stats.requests_per_second', RATE)), # < 1.8
+            ('total_xact_count',     ('pgbouncer.stats.transactions_per_second', RATE)), # >= 1.8
+            ('total_query_count',    ('pgbouncer.stats.queries_per_second', RATE)), # >= 1.8
             ('total_received',       ('pgbouncer.stats.bytes_received_per_second', RATE)),
             ('total_sent',           ('pgbouncer.stats.bytes_sent_per_second', RATE)),
             ('total_query_time',     ('pgbouncer.stats.total_query_time', GAUGE)),
-            ('avg_req',              ('pgbouncer.stats.avg_req', GAUGE)),
+            ('total_xact_time',      ('pgbouncer.stats.total_transaction_time', GAUGE)), # >= 1.8
```

Comment:
I think this should be a rate since the documentation say: 
```
Total number of microseconds spent by **pgbouncer** when connected to PostgreSQL
```

Reply:
I was copying total_query_time — if this should change, so should that. Probably makes sense that all the "total" are rates, and all the "avg" are gauges. Will update.